npm/private/npm_link_package_store.bzl

```diff
@@ -94,12 +94,8 @@ def _impl(ctx):
     files = utils.make_symlink(ctx, root_symlink_path, virtual_store_directory)
 
     for bin_name, bin_path in ctx.attr.bins.items():
-        if ctx.label.package:
-            path_to_root = "/".join([".."] * len(ctx.label.package.split("/")))
-        else:
-            path_to_root = "."
         bin_file = ctx.actions.declare_file(paths.join("node_modules", ".bin", bin_name))
-        bin_path = paths.normalize(paths.join("../..", path_to_root, virtual_store_directory.short_path, bin_path))
+        bin_path = paths.normalize(paths.join("..", package, bin_path))
```

Comment:
Did this change anything?

Reply:
yea, it changed where the generated `node_modules/.bin/foo` points to. Previously I reached into the virtual store directory but it is cleaner and more portable to run the bin entry point as `node_modules/foo/path/to/bin` that goes through the `node_modules/foo` symlink

Reply:
👍 

Did that fix anything though or just cleaner (and I think more "correct" as well)?

Reply:
It wouldn't have worked in the devserver tmp sandbox without this change since the relative path to the virtual store is not the same as from the runfiles tree.

Reply:
Comes into play with the `command` variant for specifying the tool such as
```
js_run_devserver(
    name = "serve_alt",
    args = ["{}/http_root".format(package_name())],
    command = "./node_modules/.bin/http-server",
    data = [
        ":http_root",
        "//:node_modules/http-server",
    ],
)
```